### PR TITLE
docs: update exception messages and logger statements

### LINKED ISSUE #312: Add more help text and clearer error messages to onboarding_cli
### Is your feature request related to a problem? Please describe.

While using activate_cli I found myself stuck as I was not able to figure out what was wrong after getting "Error:auth failed". It took looking at server logs to see that the activation was failing due to an issue with the cram key I was using. 

### Describe the solution you'd like

It would be nice if the tool provided more specific help text are more context in the error messages in order to help people to determine why something isn't working.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by srieteja**
closes https://github.com/atsign-foundation/at_libraries/issues/312
**- What I did**
- docs: update exception messages and logger statements for at_onboarding_cli and activate_cli

**- Description for the changelog**

docs: update exception messages and logger statements for at_onboarding_cli and activate_cli

### CODE REVIEW

File: packages/at_onboarding_cli/bin/activate_cli.dart [hunk 1]
```diff
@@ -7,8 +7,12 @@ import 'package:at_onboarding_cli/src/activate_cli/activate_cli.dart'
 Future<void> main(List<String> args) async {
   try {
     await activate_cli.main(args);
-  } on IllegalArgumentException {
+  } on IllegalArgumentException catch(e){
+    stderr.writeln('Aborting process with exit code:1 \nCause:$e');
     exit(1);
+  } catch(e){
```
Comment (murali-shris) [inline packages/at_onboarding_cli/bin/activate_cli.dart]: we should have on Exception and on Error in separate blocks

File: packages/at_onboarding_cli/lib/src/onboard/at_onboarding_service_impl.dart [hunk 2]
```diff
@@ -272,30 +283,40 @@ class AtOnboardingServiceImpl implements AtOnboardingService {
...
           exceptionScenario: ExceptionScenario.invalidValueProvided);
     }
     await _init(atKeysFileDataMap);
-    logger.finer('pkam auth');
-    _isPkamAuthenticated = (await _atLookUp?.pkamAuthenticate())!;
-    logger.finer('pkam auth result: $_isPkamAuthenticated');
+    logger.finer('Authenticating using PKAM');
+    try {
+      _isPkamAuthenticated = (await _atLookUp?.pkamAuthenticate())!;
+    } on Exception {
```
Comment (murali-shris) [inline packages/at_onboarding_cli/lib/src/onboard/at_onboarding_service_impl.dart]: Exception could be due to different reasons..catch and add e.toString to the message

Comment (srieteja) [reply packages/at_onboarding_cli/lib/src/onboard/at_onboarding_service_impl.dart]: done as part of [48707fe](https://github.com/atsign-foundation/at_libraries/pull/329/commits/48707fea63cae6a0d9c33825ec4bc37a2b22be58)

File: packages/at_onboarding_cli/bin/activate_cli.dart (hunk 1 above)
Comment (srieteja) [reply packages/at_onboarding_cli/bin/activate_cli.dart]: done as part of [48707fe](https://github.com/atsign-foundation/at_libraries/pull/329/commits/48707fea63cae6a0d9c33825ec4bc37a2b22be58)

**Review by murali-shris — APPROVED** [review]

**Pull request merged**